Jenkinsfile.deploy

```
@@ -154,6 +206,40 @@ pipeline {
                     pushImage()
                 }
             }
+            post {
+                success {
+                    script {
+                        String imgPrefix = "kogito-cloud-operator.image"
+                        // Store image deployment information
+                        setDeployPropertyIfNeeded("${imgPrefix}.registry", getDeployImageRegistry())
+                        setDeployPropertyIfNeeded("${imgPrefix}.namespace", getDeployImageNamespace())
+                        setDeployPropertyIfNeeded("${imgPrefix}.name-suffix", getDeployImageNameSuffix())
+                        setDeployPropertyIfNeeded("${imgPrefix}.tag", getDeployImageTag())
+                    }
+                }
+            }
+        }
+        stage('Create PR'){
+            when {
+                expression { return params.RELEASE }
+            }
+            steps {
+                script {
+                    githubscm.commitChanges(env.GIT_COMMITTER_NAME, env.GIT_COMMITTER_EMAIL, getCommitMessage(), ".")
```

```suggestion
                    githubscm.commitChanges(getCommitMessage(), ".")
```

I'm updating `commitChanges()` to not take Git committer parameters. See: https://issues.redhat.com/browse/KOGITO-2724